src/app/project-notifications/project-notifications.component.html

```diff
@@ -34,74 +34,73 @@ <h4>Project Notifications</h4>
                             <div class="pn-location-info">
                                 <div class="row">
                                     <div class="col-sm-12 col-md-8">
-                                        <h4>{{(project?.name || '-')}}</h4>
+                                        <h4>{{(project?.name.toUpperCase() || '-')}}</h4>
                                     </div>
                                     <div class="col-sm-12 col-md-4">
                                         <a *ngIf="project?.commentPeriod?.daysRemainingCount > 0" class="btn btn-primary view-pcp-btn"
                                         title="View this Public Comment Period"
                                         [routerLink]="['/pn', project?._id, 'cp', project?.commentPeriod?._id]">
                                         <div class="pcp-indicator pcp-indicator-active"></div>
-                                        View Open Comment Period
+                                        Open Comment Period - View/Submit
                                         </a>
                                         <a *ngIf="project?.commentPeriod?.daysRemainingCount === 0" class="btn btn-primary view-pcp-btn"
                                         title="View this Public Comment Period"
                                         [routerLink]="['/pn', project?._id, 'cp', project?.commentPeriod?._id]">
                                         <div class="pcp-indicator pcp-indicator-inactive"></div>
-                                        View {{project?.commentPeriod?.commentPeriodStatus}} Comment Period
+                                        View Completed Comment Period
                                         </a>
                                     </div>
                                 </div>
                                 <div class="row">
-                                    <div class="col-sm-12 col-md-3 pn-info-block">
+                                    <div class="pn-decision">
+                                        <a [routerLink]="['/p', project?.associatedProjectId ]" *ngIf="project?.associatedProjectId; else noHyperlink">
+                                        <p class="value">Notification Decision - {{ project?.decision }} | {{project?.decisionDate?.toString().split('T')[0] || "-" }} </p>
```

Comment:
This is showing a little off when there's '-' to say:

![image](https://user-images.githubusercontent.com/51633/106231983-cf488080-61a7-11eb-9f70-8665292fe957.png)